lib/Sema/CSApply.cpp

```diff
@@ -2501,9 +2501,9 @@ namespace {
     }
 
     // SWIFT_ENABLE_TENSORFLOW
-    /// When we've type checked a #tfop expression, we do some adjustment to
-    /// the argument and result types.  Specifically, if something is a type
-    /// that conforms to TensorProtocol (like Tensor or Tensor2D), we use the
+    /// When we've type checked a #tfop expression, we do some adjustment to the
+    /// argument types.  Specifically, if an argument is a type that conforms to
```

Comment:
Awkward wording. 

"if an argument is a type that conforms to" -> "if an argument type conforms to"

Reply:
Done